packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -3316,10 +3318,11 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
   }
 
   /// Toggles the visibility of the toolbar.
-  void toggleToolbar() {
-    assert(_selectionOverlay != null);
```

Comment:
I don't know why we didn't create the selection overlay here in the first place.  Seems fine to me.